src/contentAggregator.js

```diff
@@ -31,17 +39,18 @@ export const addContextToContent = async (dataPromise, ...dataArgs) => {
     addResumeTimeSeconds ? getResumeTimeSecondsByIds(ids) : Promise.resolve(null),
   ])
 
-  const addContext = (item) => ({
+  const addContext = async (item) => ({
     ...item,
     ...(addProgressPercentage ? { progressPercentage: progressPercentageData?.[item.id] } : {}),
     ...(addStatus ? { progressStatus: statusData?.[item.id] } : {}),
     ...(addIsLiked ? { isLiked: isLikedData?.[item.id] } : {}),
-    ...(addResumeTimeSeconds ? { resumeTime: resumeTimeData?.[item.id] } : {}),
+    ...(addLikeCount && ids.length === 1 ? { likeCount: await fetchLikeCount(item.id) } : {}),
+    ...(addResumeTimeSeconds && ids.length === 1 ? { resumeTime: resumeTimeData?.[item.id] } : {}),
```

Comment:
On second thought, this doesn't need to restriction, because `getResumTimeSecondsByIds` makes a single call for all ids it one go.
```suggestion
    ...(addResumeTimeSeconds ? { resumeTime: resumeTimeData?.[item.id] } : {}),
```